### DISCUSSION TITLE
Paste website: clear direction on the use of oppressive language

### DISCUSSION THREAD

**Discussion by travisterwilligar** [twilio-labs/paste#510] [Ideas] [answered]
We should take an opinionated stance on the use of oppressive language and propose alternatives to common oppressive terms that are unfortunately the standard in many products.

We could consider aligning with others in Twilio: https://docs.google.com/document/d/1MgSPsBo4pnjLWa5IQ_oP-71MkvhRsT4CBOV8AHij2lM/edit?ts=5ee8e77d#heading=h.2wmlzn77jhuo

**Comment by aayushpi** [comment]
Tagging @ctecks in here to get this discussion going.

**Comment by ctecks** [comment]
Fully in support of this effort, and heartened about how quickly folks have moved on it. I want to call out [Shradha's post in #team-rd-all](https://twilio.slack.com/archives/CSS1PQWS1/p1592421373135700): 

> "One request for you all: While this topic is dear to most of you, it’s important we don’t duplicate efforts across small R&D teams. 
> [Action requested] 
> 
> - If you and/ or your teams have any efforts underway already, please let us know by filling out this google form (will take < 2 mins). We’ll make this information available to everyone so you can see who’s doing what!
> 
> - If you haven’t kicked off anything yet, please do wait for the new glossary from the Council so we can stay coordinated/ aligned across R&D as well as Twilio."
 
What we can do is consider if there are any UI elements, or UX terminology that might fall into this category of oppressive language. The only one that springs to my mind is "white space", however that's a description of space that is white/clear, and I've heard it used interchangeably with "negative space". 

Interested to hear if that chimes with anyone, and if folks can identify other terminology, advice, or practices that we can and should address.

**Reply by aayushpi** [reply]
I would recommend starting with the defaults and then adding more as we see them.

**Comment by TheSisb** [comment, marked as answer]
Small update: I swapped out the usage of blacklist/whitelist in our entire codebase https://github.com/twilio-labs/paste/pull/518.  We have a backlogged ticket to remedy master/slave terminology.

**Reply by TheSisb** [reply]
Another update: we should no longer have `master/slave` terminology in our docs or codebase. Please let us know if you found any oopsies.
